pycqed/analysis_v3/data_processing.py

```diff
@@ -1,232 +1,104 @@
 import logging
```

Comment:
General comment: Good job for the documentation ! It's really nice to have clear and documented nodes :) :+1:

Reply:
I think I might be useful to make this a module with different node grouped in files, I already see this file becoming as large as the QudevTransmon when I arrived ;)

Reply:
Yep, that is the plan. I already split is into fitting, plotting, and helper functions modules. Originally I think most of those nodes were all in the data_processing.